tests/integration/conftest.py

```diff
@@ -46,6 +50,74 @@ def testing_api_key(environ: str) -> str:
     return os.environ["LABELBOX_TEST_API_KEY_STAGING"]
 
 
+def beta_endpoint(fn):
+
+    def beta(client, *args, **kwargs):
+        try:
+            client.endpoint = client.endpoint.replace("/graphql", "/_gql")
+            return fn(client, *args, **kwargs)
+        finally:
+            client.endpoint = client.endpoint.replace("/_gql", "/graphql")
+
+    return beta
+
+
+@beta_endpoint
+def cancel_invite(client, invite_id):
+    """
```

Comment:
You could maybe turn this comments into application logic through introducing a dummy decorator that annotates like test method only or something like that.

or like throw if pytest is not running:

https://stackoverflow.com/questions/25188119/test-if-code-is-executed-from-within-a-py-test-session

Reply:
The main point of the comment is to make it so that users don't copy this into their own application. I don't think there is a risk of someone importing from the conftest.